app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -272,7 +290,7 @@ export default function ManageChannelMembers({
                 selectedIds={EMPTY_IDS}
                 showManageMode={canManageAndRemoveMembers && isManageMode}
                 showNoResults={!loading}
-                term={term}
+                term={searchedTerm}
```

Comment:
This is one of the small tricks to avoid the flashing "no results" screen, since now `searchedTerm` changes at the same time as `loading`.